docs/reference/release-notes/highlights.asciidoc

```diff
@@ -1,27 +1,200 @@
 [[release-highlights]]
-= Release highlights
-
-[partintro]
---
-This section summarizes the most important changes in each release. For the
-full list, see <<es-release-notes>> and <<breaking-changes>>.
-
-* <<release-highlights-7.7.0>>
-* <<release-highlights-7.6.0>>
-* <<release-highlights-7.5.0>>
-* <<release-highlights-7.4.0>>
-* <<release-highlights-7.3.0>>
-* <<release-highlights-7.2.0>>
-* <<release-highlights-7.1.0>>
-* <<release-highlights-7.0.0>>
-
---
-
-include::highlights-7.7.0.asciidoc[]
-include::highlights-7.6.0.asciidoc[]
-include::highlights-7.5.0.asciidoc[]
-include::highlights-7.4.0.asciidoc[]
-include::highlights-7.3.0.asciidoc[]
-include::highlights-7.2.0.asciidoc[]
-include::highlights-7.1.0.asciidoc[]
-include::highlights-7.0.0.asciidoc[]
+== What's new in {minor-version}
+
+Here are the highlights of what's new and improved in {es} {minor-version}! 
+For detailed information about this release, see the 
+<<release-notes-{elasticsearch_version}, Release notes >>  and 
+<<breaking-changes-{minor-version}, Breaking changes>>.
+
+// Add previous release to the list
+Other versions: 
+{ref-bare}/7.7/release-highlights.html[7.6]
+| {ref-bare}/7.5/release-highlights-7.5.0.html[7.5]
+| {ref-bare}/7.4/release-highlights-7.4.0.html[7.4]
+| {ref-bare}/7.3/release-highlights-7.3.0.html[7.3]
+| {ref-bare}/7.2/release-highlights-7.2.0.html[7.2]
+| {ref-bare}/7.1/release-highlights-7.1.0.html[7.1]
+| {ref-bare}/7.0/release-highlights-7.0.0.html[7.0]
+
+// tag::notable-highlights[]
```

Comment:
I usually just put one tag at the top of the file and another at the bottom.

Reply:
Due to the large number of things that end up in the ES highlights, they aren't necessarily all featured in the stack highlights.